packages/atlas/src/components/_inputs/FileSelect/FileSelect.tsx

```diff
@@ -37,9 +36,9 @@ export type FileSelectProps = {
   isFileLoading?: boolean
   onReAdjustThumbnail?: () => void
   onDropRejected?: (fileRejections: FileRejection[]) => void
-  onError?: (error: string | null, fileType: FileType) => void
   error?: string | null
   maxSize?: number
+  setError: Dispatch<SetStateAction<string | null>>
```

Comment:
I would keep the `onError` name. Also `Dispatch<SetStateAction<string | null>>` pretty much the same as `(error: string | null) => void`, and the latter is more readable to me to be honest.